fireplace/actions.py

```diff
@@ -194,7 +194,7 @@ def do(self, source, attacker, defender):
 		defender.defending = True
 		source.game.proposed_attacker = attacker
 		source.game.proposed_defender = defender
-		source.game.manager.step(Step.MAIN_COMBAT, Step.MAIN_ACTION)
+		source.game.manager.step(Step.MAIN_COMBAT, source.game.step)
```

Comment:
I disagree. The expectation is that we go into MAIN_ACTION after we're done with combat.

If this line isn't actually useful to "fix the MAIN_COMBAT post-step", then it should be left out anyhow.